LLM/app/utils/DB/faissDB.py

```diff
@@ -3,20 +3,28 @@
 import numpy as np
 import pickle
 import os
+from googletrans import Translator
```

Comment:
[из доки](https://pypi.org/project/googletrans/):

> You may wonder why this library works properly, whereas other approaches such like goslate won’t work since Google has updated its translation service recently with a ticket mechanism to prevent a lot of crawler programs.
> 
> I eventually figure out a way to generate a ticket by reverse engineering on the [obfuscated and minified code used by Google to generate such token](https://translate.google.com/translate/releases/twsfe_w_20170306_RC00/r/js/desktop_module_main.js), and implemented on the top of Python. However, this could be blocked at any time.

Reply:
> а ты уверен, что эта библиотека/модель достаточно хорошо умеет специализированный, технический текст? Я имею в виду, что у нас текст состоит из терминов, точно ли она все переведет без потери смысла?

Оно переводит для эмбедингов, т.е. одинаковые термины будут иметь одинаковый перевод и одинаковый эмбединг, плюс ещё с английским модель лучше работает (для поиска близких по смыслу, в БЯМ передаём по прежнему на русском)

Reply:
Ну, 1ое, посмотри коммент Саши, нам в любой момент это могут отрубить.
Второе - мне кажется, что одинаковые термины не факт, что будут иметь одинаковый перевод, переводчик в зависимости от контекста может что-то поменять, по идее близость таких слов будет большая, но не знаю)

Насколько лучше работает и нужно ли это нам, ты тестил именно на наших данных, желательно без текстов про велосипеды и Париж

Reply:
я проверил, на русском работает хуже, чем на переведённом английском, одинаковые термины на русском точно также могут иметь разные эмбединги в разном контексте 

Если отрубят, будем фиксить, что можно быстро сделать) Эта либа позволяет использовать не локальные ресурсы, что плюс к производительности

Reply:
Поч нельзя пофиксить сейчас?

Reply:
тогда будут задействованы локальные ресурсы, что ухудшит производительность

Reply:
Давай хотя бы инкапсулируешь это все в класс переводчик, так будет легче переделывать)

Reply:
добавил, но не думаю, что google отключит либу до конца сема и даже в ближайшие пару лет...

Reply:
> добавил, но не думаю, что google отключит либу до конца сема и даже в ближайшие пару лет...
> 
> так что в планах нет переделывания

Либа поломается, если гугл поменяет апи или изменит алгоритм генерации токена